lib/iputil.js

```diff
@@ -35,3 +40,22 @@ exports.maskLeft = (cleanXffString) => {
   parts[0] = exports.mask(parts[0]);
   return parts.join(', ');
 }
+
+/**
+ * Convert to a fixed length string
+ */
+exports.fixed = (cleanIpString) => {
+  return exports.fixedKind(cleanIpString)[0];
+}
+exports.fixedKind = (cleanIpString) => {
+  if (ipaddr.isValid(cleanIpString)) {
+    const ip = ipaddr.parse(cleanIpString);
+    if (ip.kind() === 'ipv4') {
+      return [ip.octets.map(n => `00${n}`.substr(-3, 3)).join('.'), 'v4'];
```

Comment:
I see, normalizing the addresses by padding with 0s, works for both v4 and v6, but there is a `toFixedLengthString` for the v6.

Reply:
Yep - not sure why they have a built-in method for one but not the other.  Pretty simple to DIY though.